src/generic/grid.cpp

```diff
@@ -3832,7 +3901,7 @@ void wxGrid::ProcessRowLabelMouseEvent( wxMouseEvent& event, wxGridRowLabelWindo
                     if ( !m_selection->IsInSelection(m_currentCellCoords) )
                         break;
 
-                    if ( (row = YToRow( pos.y )) >= 0 )
+                    if (row >= 0 )
```

Comment:
Even more trivial, but let's keep spacing consistent
```suggestion
                    if ( row >= 0 )
```